lib/Importer/Caffe2.cpp

```diff
@@ -454,7 +465,18 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     assert(!transLHS && "Don't support transpose lhs for now.");
     bool transRHS = dict.count("trans_b") && (loadInt(dict["trans_b"]) == 1);
     if (transRHS) {
-      RHS = G_.createTranspose("RHS.transpose", RHS, {1, 0});
+      // The semantic of the transpose in that context is:
+      // swap the last two dimensions.
+      unsigned_t nbDims = RHS.dims().size();
+      assert(nbDims >= 2 && "C2 specs say rank of RHS must be >= 2");
+      std::vector<unsigned_t> shuffle;
+      unsigned_t i;
+      for (i = 0; i < nbDims - 2; ++i) {
```

Comment:
In release mode, if assertion fails, this loop may turn into time & memory eating thing. May be annoying to fight with on local machine.

Reply:
What's the proper way to report an error in the loader?
The loadOperator method does not return any status.

GLOW_ASSERT?

Reply:
Went with GLOW_ASSERT.